odl/solvers/advanced/proximal_operators.py

```diff
@@ -31,7 +31,7 @@
 from builtins import super
 
 import numpy as np
-import scipy.special
+import scipy
```

Comment:
What version of scipy are you using?

Scipy uses lazy imports (you may need to google what it means), and thus *if* `special` is imported lazily, it will not be available via `scipy.special.lambertw`, and you need to do `import scipy.special`. There is some text on this [here](http://docs.scipy.org/doc/scipy/reference/api.html), but it's rather vague.